docs/Makefile

```
@@ -16,12 +16,12 @@ help:
 
 .PHONY: help Makefile generate-docs
 
-$(ANSIBLEGIT):
-	git clone --depth=1 https://github.com/ansible/ansible/ -b stable-2.9 $(ANSIBLEGIT)
-
-generate-docs: $(ANSIBLEGIT)
-	rm -rf ./modules/
-	bash -c "source $(ANSIBLEGIT)/hacking/env-setup && ANSIBLE_COLLECTIONS_PATHS=../build/collections python3 $(ANSIBLEGIT)/hacking/build-ansible.py document-plugins --module-dir ../plugins/modules --template-dir ./_templates --template-dir $(ANSIBLEGIT)/docs/templates --type rst --output-dir ./modules/"
+generate-docs:
+	mkdir -p ./plugins/
+	rm -rf ./plugins/*
```

```suggestion
	rm -rf ./plugins/
	mkdir -p ./plugins/
```
That way we even remove .* files.

---

Interesting. I had it like this before and then thought "it feels wrong to delete and create the main directory all the time". But of course hidden files are a good point.

---

See my other comment. To me it feels wrong to delete files at all to recreate them in the same Make snippet.